fastlane/lanes/release.rb

```diff
@@ -280,3 +290,15 @@ def report_milestone_error(error_title:)
 
   buildkite_annotate(style: 'warning', context: 'error-with-milestone', message: error_message) if is_ci
 end
+
+def delete_all_metadata_release_notes(store_metadata_folder: STORE_METADATA_FOLDER)
+  files = Dir.glob(File.join(store_metadata_folder, '**', 'release_notes.txt'))
+  files.each { |path| File.delete(path) }
+  git_add(path: files)
+  git_commit(
+    path: files,
+    message: 'Delete release notes source and localization before code freeze',
```

Comment:
nitpick for a future PR:

```suggestion
    message: 'Delete release notes of previous versions before code freeze'
```